crypto/provider_conf.c

```diff
@@ -244,7 +244,11 @@ static int provider_conf_load(OSSL_LIB_CTX *libctx, const char *name,
         else if (strcmp(confname, "module") == 0)
             path = confvalue;
         else if (strcmp(confname, "activate") == 0)
-            activate = 1;
+            if ((strcmp(confvalue, "1") == 0)
+                || (strcmp(confvalue, "yes") == 0)
+                || (strcmp(confvalue, "true") == 0)
+                || (strcmp(confvalue, "on") == 0))
+                activate = 1;
```

Comment:
I think this change probably justifies an update to CHANGES.md

Reply:
done